index.bs

```diff
@@ -1951,6 +2034,149 @@ The [=remote end steps=] with |session| and <var ignore>command parameters</var>
 
 </div>
 
+#### The browser.createUserContext Command #### {#command-browser-createUserContext}
+
+The <dfn export for=commands>browser.createUserContext</dfn> command creates a
+[=user context=].
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+      <pre class="cddl remote-cddl">
+      browser.CreateUserContext = (
+        method: "browser.createUserContext",
+        params: EmptyParams,
+      )
+      </pre>
+   </dd>
+   <dt>Return Type</dt>
+   <dd>
+    <pre class="cddl local-cddl">
+      browser.createUserContextResult = {
+        userContext: browser.UserContext
+      }
```

Comment:
Shouldn't it actually re-use the already existent type definition?

```suggestion
      browser.createUserContextResult = (
        browser.UserContextInfo
      )
```

Reply:
Good point, updated the struct and also the text.